.travis.yml

```diff
@@ -11,15 +11,27 @@ language: ruby
 services:
   - docker
 
+# Make sure the instances listed below match up with
+# the `platforms` defined in `kitchen.yml`
 env:
   matrix:
-    - INSTANCE: debian-9
-    - INSTANCE: debian-8
-    - INSTANCE: ubuntu-1804
-    - INSTANCE: ubuntu-1604
-    - INSTANCE: centos-7
-    - INSTANCE: fedora
-    - INSTANCE: opensuse-42
+    - INSTANCE: default-debian-9-2019-2-py3
+    - INSTANCE: default-ubuntu-1804-2019-2-py3
+    - INSTANCE: default-centos-7-2019-2-py3
+    - INSTANCE: default-fedora-29-2019-2-py3
+    - INSTANCE: default-opensuse-leap-15-2019-2-py3
+    - INSTANCE: default-debian-9-2018-3-py2
+    - INSTANCE: default-ubuntu-1604-2018-3-py2
+    - INSTANCE: default-centos-7-2018-3-py2
+    - INSTANCE: default-fedora-29-2018-3-py2
+    # TODO: Use this when fixed instead of `opensuse-leap-42`
+    # Ref: https://github.com/netmanagers/salt-image-builder/issues/2
+    # - INSTANCE: default-opensuse-leap-15-2018-3-py2
+    - INSTANCE: default-opensuse-leap-42-2018-3-py2
+    - INSTANCE: default-debian-8-2017-7-py2
+    - INSTANCE: default-ubuntu-1604-2017-7-py2
+    - INSTANCE: default-fedora-28-2017-7-py2
+    - INSTANCE: default-opensuse-leap-42-2017-7-py2
```

Comment:
Please use the reduced matrix merged into the `template-formula` recently, or similar.